ext/TrixiMakieExt.jl

```diff
@@ -410,6 +404,60 @@ function Makie.plot!(fig, pd::PlotData2DTriangulated;
 
     return FigureAndAxes(fig, axes)
 end
-end # @muladd
 
+# converts a single int into a tuple of ints, to get a square arrangement
+# example: f(1) = (1,1) f(2) = (2,1) f(3) = (2,2) f(4) = (1,2)
+function makie_layout_helper(n)
+    if n == 1
+        return (1, 1)
+    end
+    t = makie_layout_helper(n - 1)
+    if t[1] == 1
+        return (t[2] + 1, 1)
+    elseif t[1] > t[2]
+        return (t[1], t[2] + 1)
+    elseif t[2] >= t[1]
+        return (t[1] - 1, t[2])
+    end
+end
```

Comment:
Can you please comment on what the input and output mean? My first guess was

> Given `n` plots, how many rows and columns shall I use?

However, this does not fit `f(4) = (1,2)`, since this would mean there are only 2 plots...